client/src/app/pages/migration-waves/components/migration-wave-form.tsx

```diff
@@ -138,15 +138,7 @@ export const WaveForm: React.FC<WaveFormProps> = ({
     name: yup
       .string()
       .trim()
-      .min(3, t("validation.minLength", { length: 3 }))
-      .max(120, t("validation.maxLength", { length: 120 }))
-      .test(
-        "Duplicate name",
-        "An identity with this name already exists. Use a different name.",
```

Comment:
Maybe we should keep this for duplicate migration wave names.

Reply:
@ibolton336, I thought about it but it isn't a requirement.

Reply:
Seems like an implied requirement that we wouldn't want a user creating duplicates IMO. Is there a case FOR having duplicate migration waves? 

<img width="1906" alt="Screenshot 2023-06-12 at 3 55 36 PM" src="https://github.com/konveyor/tackle2-ui/assets/11218376/5fb5eca7-b56e-49e5-be5c-835660aff437">

Reply:
@ibolton336, I don't think there is any implied requirement.
Maybe a migrator would like to break down a migration in separate waves starting at different times.
@rromannissen, what do you think ?

Reply:
Let's follow-up with core team to have a composite primary key.

Reply:
> I don't think there should be duplicate waves.
So sounds like we should just open up a new issue and add the duplicate check back in?

Reply:
@ibolton336, the duplicate check will need to include start and end dates.
And make sure the hub enforces that too.

Reply:
Gotcha. So the name field is set to start date/ end date when no name exists, right? So wouldn't the existing check work for that, too?